sdk/ai/azure-ai-projects/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Features added
 
+* Add the instructions on how to deploy an Azure functions into RREADME.
```

Comment:
Typo RREADME (and I would say "package README.md")

Reply:
Fixed.